docs/src/main/sphinx/release/release-440.md

```diff
@@ -63,6 +63,10 @@
 
 ## Iceberg connector
 
+```{warning}
+This release has a major regression which is fixed in Release 442.
```

Comment:
```suggestion
This release has a major regression which is fixed in Trino 442.
```

also do we need the same in 441? 

I am not sure we need this at all .. did we do this in the past?

Reply:
Yeah, we've added a warning for major regressions that required emergency releases to patch. I don't think we need it in 441 though